### DISCUSSION TITLE
sub folder install : glide failing

### DISCUSSION THREAD

**Discussion by breizhwave** [statamic/cms#5632] [Troubleshooting]
Hi I get 404 on glide images on a statamic installed in subfolder, have looked at every possible configuration tweaks without success, any idea ? 

http://localhost:8888/statamix/lizzJs3/public/asset/img/YXNzZXRzL21vdXRhaW4uanBlZw==?h=570&w=736&fit=crop&s=774b19f6b530b84715ddde49e695287b

my setup is the following : 
**.env**
`APP_URL=http://localhost:8888/statamix/lizzJs3/public/`
**app.php**
```
    'url' => env('APP_URL', 'http://localhost:8888/statamix/lizzJs3/public/'),

    'asset_url' => env('ASSET_URL', 'http://localhost:8888/statamix/lizzJs3/public/'),
```
**sites.php**
```
 'sites' => [
        'default' => [
            'name' => config('app.name'),
            'locale' => 'en_US',
            'url' => 'http://localhost:8888/statamix/lizzJs3/public/',
        ],

    ],
 
```
did run the following
```
php artisan storage:link
The [/Applications/MAMP/htdocs/statamix/lizzJs3/public/storage] link has been connected to [/Applications/MAMP/htdocs/statamix/lizzJs3/storage/app/public].
The links have been created.
```

**Comment by edalzell** [comment]
I don't think those paths are right. `public` is never included in any "url" type configs.

Also, does plain Laravel work? I think something needs to be done to make Laravel work in a sub folder, have you done those steps?

**Reply by breizhwave** [reply]
thanks for your answer... did all I thought was necessary and everything works fine including upload of images, it's just the glide that must be missing some cache folder related configuration... It's just for testing anyway I know real installs are on absolute folders but still might be itnersting to duplicate installations on subfolders... even though that's getting closer the PRO features we can all afford too contribute. marvelous CMS anyway
